my_sql.md

```diff
@@ -0,0 +1,37 @@
+# MySQL
+
+
+## Prerequisite knowledge:   
+DATABASES
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+* I know what kind of database MySQL is and know it's pros and cons
+* I can create a database
+* I can view database structure with basic commands
+* I know MySQL data types
+* I can create, update and delete records in the database
+
+#### Intermediate aka _I can use it_
+* I can use constraints in database models: not null, unique, primary key, foreign key, default and index
+* I can list records meeting specified conditions using various filters (e.g. where, having, group by, order by, limit)
+* I can use joins to combine rows from different tables
+* I can use aggregate functions
+
+#### Upper-Intermediate aka _upper-intermediate_
```

Comment:
I think "usage and understanding of the `explain` statement" should be placed at this level.